### PR TITLE
[stable23] Run audit

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by nickvergessen** [comment]
I don't know how to fix this ... Would not have assumed that npm audit fix can do something like this.

**Comment by PVince81** [comment]
@skjnldsv can you help sort this out ?

**Comment by skjnldsv** [comment]
> I don't know how to fix this ... Would not have assumed that npm audit fix can do something like this.

did you audit force?
Not all audits can be fixed without a tremendous amount of changes.

File: package.json [hunk 1]
```diff
@@ -59,13 +59,13 @@
     "stylelint-webpack-plugin": "^2.2.2",
     "vue-loader": "^15.9.8",
     "vue-template-compiler": "^2.6.14",
-    "webpack": "^4.46.0",
+    "webpack": "^5.74.0",
```
Comment (skjnldsv) [inline package.json:62]: Updating a major dep in a stable branch is a no-go from me

**Comment by blizzz** [comment]
:ping_pong:

**Comment by nickvergessen** [comment]
Will close this, next release is the last one anyway

**Pull request closed without merging**